languages/csharp/AppEncryption/README.md

```diff
@@ -224,63 +95,146 @@ Dictionary<string, string> regionDictionary = new Dictionary<string, string>
 
 // Build the Key Management Service using the region dictionary and your preferred (usually current) region
 AWSKeyManagementServiceImpl keyManagementService = AWSKeyManagementServiceImpl.newBuilder(regionDictionary, "us-east-1").build();
+```
 
-// Provide the above keyManagementService to the session factory builder
-using (AppEncryptionSessionFactory appEncryptionSessionFactory = AppEncryptionSessionFactory.NewBuilder("productId", "systemId")
-     .WithMetastorePersistence(metastorePersistence)
-     .WithCryptoPolicy(policy)
-     .WithKeyManagementService(keyManagementService)
-     .Build()) {
+#### Static KMS (FOR TESTING ONLY)
 
-    // ...
-}
+```c#
+KeyManagementService keyManagementService = new StaticKeyManagementServiceImpl("secretmasterkey!");
 ```
 
-### Crypto Policy
-Please refer to the [Java lib's notes](../../java/app-encryption#crypto-policy) until documentation cleanup/refactor is complete. The only difference code-wise is the convention of using PascalCase instead of camelCase for method naming.
+### Define the Crypto Policy
+Detailed information on Crypto Policy can be found [here](../../../docs/CryptoPolicy.md). The Crypto Policy's effect 
+on key caching is explained [here](../../../docs/KeyCaching.md).
 
-### Key Caching
-Please refer to the [Java lib's notes](../../java/app-encryption#key-caching) until documentation cleanup/refactor is complete. The only difference code-wise is the directory path to the C# classes, but semantics should be the same.
+#### Basic Expiring Crypto Policy
 
-### Metrics
-The library uses [App.Metrics](https://www.app-metrics.io/) for metrics, which are disabled by default. If metrics are left disabled, we simply create and use an `IMetrics`instance whose [Enabled flag](https://www.app-metrics.io/getting-started/fundamentals/configuration/) is disabled.
+```c#
+CryptoPolicy cryptoPolicy = BasicExpiringCryptoPolicy
+    .NewBuilder()
```

Comment:
nit: can put the `NewBuilder()` on previous line (i.e. `CryptoPolicy cryptoPolicy = BasicExpiringCryptoPolicy.NewBuilder()`) -- ditto on java readme if possible